packages/manager/modules/pci/src/projects/project/storages/object-storage/users/users.html

```diff
@@ -0,0 +1,65 @@
+<cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+
+<div data-ng-if="$ctrl.userList.length === 0">
+    <h5
+        data-translate="pci_projects_project_storages_containers_users_user_management"
+    ></h5>
+    <p
+        data-translate="pci_projects_project_storages_containers_users_user_description"
+    ></p>
+</div>
+
+<h5
+    data-ng-if="$ctrl.userList.length > 0"
+    data-translate="pci_projects_project_storages_containers_users_title"
+></h5>
+
+<oui-datagrid data-rows="$ctrl.userList">
+    <oui-datagrid-column
+        data-title="'pci_projects_project_storages_containers_users_username' | translate"
+        data-property="username"
+        data-type="string"
+        data-searchable
+        data-sortable="asc"
+        data-filterable
+    >
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title="'pci_projects_project_storages_containers_users_description' | translate"
+        data-property="description"
+        data-type="string"
+        data-sortable
+        data-filterable
+    ></oui-datagrid-column>
+    <oui-action-menu data-compact data-placement="end">
+        <oui-action-menu-item data-on-click="$ctrl.goToImportPolicy($row)">
+            <span
+                data-translate="pci_projects_project_storages_containers_users_import_json"
+            ></span>
+        </oui-action-menu-item>
+        <oui-action-menu-item data-on-click="$ctrl.downloadJson($row)">
+            <span
+                data-translate="pci_projects_project_storages_containers_users_download_json"
+            ></span>
+        </oui-action-menu-item>
+        <oui-action-menu-item data-on-click="$ctrl.goToDeleteUser($row)">
+            <span
+                data-translate="pci_projects_project_storages_containers_delete_label"
+            ></span>
+        </oui-action-menu-item>
+    </oui-action-menu>
+    <oui-datagrid-topbar>
+        <button
+            class="oui-button oui-button_secondary"
+            data-ng-click="$ctrl.goToUsersAndRoles()"
+            data-track-on="click"
+            data-track-name="PCI_PROJECTS_CONTAINERS_ADD"
+            data-track-type="navigation"
+        >
+            <i class="oui-icon oui-icon-add pr-1" aria-hidden="true"></i>
```

Comment:
Following change introduced here `packages/manager/modules/pci/src/projects/project/storages/containers/containers.html`, would be neat to replace the `<i>` element by a `<span>`.